cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -855,4 +857,81 @@ public boolean apply(CliArguments arguments, Commandlet cmd, CompletionCandidate
     return true;
   }
 
+  public Path findBashPath() {
+
+    Path bashPath;
+    if (systemInfo.isLinux() || systemInfo.isMac()) {
+      bashPath = Path.of("/bin/bash");
```

Comment:
The existing behavior was to find `bash` on the `PATH` as not all unix variants have it in `/bin/`:
```suggestion
      bashPath = Path.of("bash");
```

As this will currently raise an invalid warning, we would need to either only do the warning in the Windows case or we actually should consider to find it from the `PATH` here.